src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.ts

```diff
@@ -79,10 +80,14 @@ export class TimestampAcceptEditorComponent implements OnInit {
     this.timestampResponseStatus = this.config.data.timestampResponseStatus
     this.ports = this.config.data.ports;
     this.updateTerminalOptions(this.transportCall.UNLocationCode);
-    this.setDefaultTimestampValues();
     this.translate.onLangChange.subscribe((event: LangChangeEvent) => {
     });
-
+    this.timestampFormGroup = this.formBuilder.group({
+      vesselPositionLongitude: new FormControl(null, [Validators.pattern("^[0-9.]*$"), Validators.maxLength(11)]),
+      vesselPositionLatitude: new FormControl(null, [Validators.pattern("^[0-9.]*$"), Validators.maxLength(10)]),
+      milesToDestinationPort: new FormControl(null, [Validators.pattern('^[0-9]+(.[0-9]{0,1})?$')]),
+    });
```

Comment:
How come we are using this style of validation for these rather than the template ones?

If it is the length, then it can be coded into the pattern via `"^[0-9.]{,11}$"`